### PR TITLE
add SECURITY_POST_LOGIN_ALLOW_REDIRECT_DOMAIN option

### PR DESCRIPTION
**Pull request by wong2**
Consider this: after login from `www.domain.com`, the user should be redirected to `en.domain.com`, which is not allowed by Flask-Security currently.

so a new option `SECURITY_POST_LOGIN_ALLOW_REDIRECT_DOMAIN` is introduced, in the above situation, one could set `SECURITY_POST_LOGIN_ALLOW_REDIRECT_DOMAIN = '*.domain.com'` to allow such redirection.

### CODE REVIEW

**Comment by wong2** [comment]
ping @mattupstate

File: docs/configuration.rst [hunk 1]
```diff
@@ -314,6 +314,11 @@ Miscellaneous
 ``SECURITY_DATETIME_FACTORY``                 Specifies the default datetime
                                               factory. Defaults to
                                               ``datetime.datetime.utcnow``.
+``POST_LOGIN_ALLOW_REDIRECT_DOMAIN``          Specifies domain allowed to be
```
Comment (jirikuncar) [inline docs/configuration.rst:317]: Please use `SECURITY_` prefix.

File: docs/configuration.rst [hunk 2]
```diff
@@ -314,6 +314,11 @@ Miscellaneous
...
+                                              redirected to after login. You can
+                                              use wildcard patterns like 
+                                              ``*.doumail.com``. Defaults to the
```
Comment (jirikuncar) [inline docs/configuration.rst:320]: ``*.doumail.com`` -> ``*.example.com`` to be clear that it's an example.

**Comment by wong2** [comment]
sorry it has been too long, can't remember the details here

**Pull request closed without merging**